vector/src/main/res/xml/vector_settings_labs.xml

```diff
@@ -44,11 +44,6 @@
         android:key="SETTINGS_LABS_UNREAD_NOTIFICATIONS_AS_TAB"
         android:title="@string/labs_show_unread_notifications_as_tab" />
 
-    <im.vector.app.core.preference.VectorSwitchPreference
-        android:defaultValue="false"
-        android:key="SETTINGS_LABS_USE_RESTRICTED_JOIN_RULE"
-        android:summary="@string/labs_use_restricted_join_rule_desc"
-        android:title="@string/labs_use_restricted_join_rule" />
```

Comment:
This removal will need more cleanup in the code, and not really related to this PR.

Reply:
Agreed will restore and creating separate PR

Reply:
PR up here https://github.com/vector-im/element-android/pull/4889